docs/getting-started-logging.md

```diff
@@ -0,0 +1,75 @@
+| +------------------------------------+                                                           |
+| |             loki-stack             |                                                           |
+| |                                    |                                       +-----------------+ |
+| | +--------------+  +--------------+ | ---------------------(Logging)------> | Scalar Products | |
+| | |     Loki     |  |   Promtail   | |                                       |                 | |
+| | +--------------+  +--------------+ |                                       |  +-----------+  | |
+| +------------------------------------+                                       |  | Scalar DB |  | |
+|                                                                              |  +-----------+  | |
+| +------------------------------------------------------+                     |                 | |
+| |                kube-prometheus-stack                 |                     |  +-----------+  | |
+| |                                                      |                     |  | Scalar DL |  | |
+| | +--------------+  +--------------+  +--------------+ | ---(Monitoring)---> |  +-----------+  | |
+| | |  Prometheus  |  | Alertmanager |  |   Grafana    | |                     +-----------------+ |
+| | +-------+------+  +------+-------+  +------+-------+ |                                         |
+| |         |                |                 |         |                                         |
+| |         +----------------+-----------------+         |                                         |
+| |                          |                           |                                         |
+| +--------------------------+---------------------------+                                         |
+|                            |                                                                     |
+|                            |         Kubernetes (minikube)                                       |
++----------------------------+---------------------------------------------------------------------+
+                             | <- expose to localhost (127.0.0.1)
+                             |
+              (Access Dashboard through HTTP)
+                             |
+                        +----+----+
+                        | Browser |
+                        +---------+
+```
+
+## Step 1. Prepare a custom values file
+
+1. Get the sample file [scalar-loki-stack-custom-values.yaml](./conf/scalar-loki-stack-custom-values.yaml) for `loki-stack`.
+
+## Step 2. Deploy `loki-stack`
+
+1. Add the `grafana` helm repository.
+   ```console
+   helm repo add grafana https://grafana.github.io/helm-charts
+   ```
+
+1. Deploy the `loki-stack`.
+   ```console
+   helm install scalar-logging-loki grafana/loki-stack -n monitoring -f scalar-loki-stack-custom-values.yaml
+   ```
+
+## Step 3. Access Dashboards
```

Comment:
the Dashboards? or the dashboard?
(do we need `the`, should it be a plural or singular?

Reply:
I think maybe we can use `Access the Grafana dashboard` here.
Let me revise it.

Reply:
Revised in https://github.com/scalar-labs/helm-charts/pull/87/commits/dca34373ad8ae165792e9ed3a5bcac6d17af128d